src/app/containers/Brand/index.stories.jsx

```diff
@@ -9,11 +9,12 @@ storiesOf('Containers|Brand', module)
   .addParameters({ chromatic: { disable: true } })
   .addDecorator(withKnobs)
   .addDecorator(withServicesKnob())
-  .add('default', ({ service }) => {
+  .add('default', ({ service, variant = 'default' }) => {
```

Comment:
I thought it would be better to have this in `psammead-storybook-helpers`.  Opened a PR to add a default to `variant` https://github.com/bbc/psammead/pull/2704

Reply:
thanks Isaac 💯